doc-content/enterprise-only/employee-rostering/wb-employee-rostering-using-rest-proc.adoc

```diff
@@ -0,0 +1,10 @@
+[id='wb-employee-rostering-using-rest-proc']
+= Accessing the solver using the REST API
+
+After deploying or re-creating the sample solver, you can access it using the REST API.
+
+You must register a solver instance using the REST API. Then you can supply data sets and retrieve optimized solutions.
+
+.Prerequisite
+
+* The employee rostering project has been set up and deployed according to the previous sections in this document. You can either deploy the sample project, as described in <<wb-employee-rostering-example-deploying-in-central-proc>>, or re-create the project, as described in <<wb-employee-rostering-example-recreate-con>>. 
```

Comment:
"The employee rostering project is set up and deployed .." instead of "The employee rostering project has been set up and deployed ..."